.github/workflows/dotnet-core.yml

```diff
@@ -0,0 +1,28 @@
+name: .NET Core
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup .NET Core
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 2.2.108
```

Comment:
Shouldn't this be 3.1? I thought 2.2 is already deprecated by MS.

Reply:
We are still at `netstandard2.0`
from https://docs.microsoft.com/en-us/dotnet/standard/net-standard, .net core 3.0 supports 2.1, so I think the lastest version `dotnet-version: 3.1.301` won't work

Do we want to upgrade to `netstandard2.1`?

Reply:
I see. No, let's stay on netstandard2.0